### PR TITLE
Global module with Afterpay brand

### PR DESCRIPTION
**Pull request by rmarinleal**

### CODE REVIEW

File: Documentation/configuration.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+In the Pagantis admin panel, we can set the following options:
+
+| Field &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;| Description<br/><br/>
+| :------------- |:-------------| 
+| Module is enabled    | * Yes => Activates the payment method (Default) <br/> * No => Disables the payment method
+| Public Key(*) |  String.
+| Secret Key(*) |  String. 
+| Simulator is enabled |  * Yes => Display the installments simulator  (Default) <br/> * No => Do not display the simulator
+
+:information_source: - Your keys are located on your [Pagantis profile](https://merchant.afterpay.com/)
```
Comment (tmeliotpg) [inline Documentation/configuration.md]: https://developers.afterpay.com/

File: afterpay.php [hunk 2]
```diff
@@ -0,0 +1,1340 @@
...
+            $templateConfigs['TITLE'] = (string) $checkoutText;
+            $language = Language::getLanguage($this->context->language->id);
+            if (isset($language['locale'])) {
+                $language = $language['locale'];
+            } else {
+                $language = $language['language_code'];
+            }
+
+            $templateConfigs['ISO_COUNTRY_CODE'] = str_replace('-', '_', $language);
+            if ($templateConfigs['ISO_COUNTRY_CODE'] == 'ca_ES') {
```
Comment (tmeliotpg) [inline afterpay.php:352]: This is not a valid language tag, it is very important to use valid tags in a global context.
```php
 public static function getFormattedLocaleTag()
    {
        $languageCode = str_replace('-','_',Context::getContext()->language->language_code);
        $languageTag = Tools::substr($languageCode, 0, 2);
        $geographicTag = Tools::strtoupper( Tools::substr($languageCode, 3, 5));
        $formattedLanguageTag =  $languageTag . '_'. $geographicTag;
        return $formattedLanguageTag;
    }
    `

File: Documentation/configuration.md (hunk 1 above)
Comment (rmarinleal) [reply Documentation/configuration.md]: solved

File: afterpay.php (hunk 2 above)
Comment (rmarinleal) [reply afterpay.php:352]: we can review it on the next release, right now is working properly and validated by QA Team

**Pull request merged**